crates/core/tedge_mapper/src/core/converter.rs

```diff
@@ -67,6 +67,8 @@ pub trait Converter: Send + Sync {
     fn sync_messages(&mut self) -> Vec<Message> {
         vec![]
     }
+
+    fn process_operation_update_messages(&mut self, _message: &Message) {}
```

Comment:
This signature of this function must be similar to `try_init_messages` i.e returning messages to be sent on update.